### DISCUSSION TITLE
Delete multiple recipes or undo migration import

### DISCUSSION THREAD

**Discussion by ghost** [mealie-recipes/mealie#1965] [Feature Request]
It would be helpful to be able to bulk-delete multiple recipes. For example, I am trying to re-import recipes from Paprika, but after importing the export file, all recipes are duplicated. The only way I can figure out how to fix this is to delete recipes one by one, or delete my Mealie volume and rebuild.

**Comment by michael-genson** [comment]
You can do this from the Manage Data menu (`https://your-mealie-url.com/group/data/recipes`)

**Comment by kramttocs** [comment]
This does work though would be nice to add some searching/filter to the page.
For example if a link was imported that isn't really a recipe it'd be nice to filter on something like:  "Could not detect instructions"

Right now ordering by servings and looking at the 0 records is as close as I am finding.
